home/npc_movement.asm

```diff
@@ -50,10 +50,12 @@ RunNPCMovementScript::
 EndNPCMovementScript::
 	farjp _EndNPCMovementScript
 
-DebugPressedOrHeldB::
+DebugBButtonToggle:: ; dummy (non-debug)
```

Comment:
This routine isn't really "toggle-ing" anything. I think the original name was probably better imo.

Reply:
How about something like `DebugBButtonSkip`?

Reply:
It returns `nz` iff the B button is held or pressed in debug mode. Which is exactly what the old name said.

Reply:
I'll revert the change.